src/meshio/netgen/__init__.py

```diff
@@ -0,0 +1,3 @@
+from ._netgen import read, write
```

Comment:
We need a link the the format specification as a top comment.

Reply:
Unfortunately there is no official documentation to the format, does a link to the implementation work too?
https://github.com/NGSolve/netgen/blob/master/libsrc/meshing/meshclass.cpp#L430

Reply:
as far as I remember, there were not that many open formats for arbitrary order curved elements, back in the late 90s ...

anyway, Netgen format in meshio would be very helpful for quite some people using Netgen + NGSolve, and future support is provided by Matthias and his colleagues 

if you need it, you get the link to the docu, no problem

thanks a lot for your nice tool

JS

Reply:
Interesting. Perhaps netgen can look at supporting other file formats in the future. At least VTK and XDMF support curved elements and have deep ecosystem support.